internal/jennies/java/tools.go

```diff
@@ -23,6 +23,14 @@ func formatScalar(val any) any {
 	return newVal
 }
 
+func cleanString(s string) string {
```

Comment:
This function could use a more descriptive name: "clean string" feels pretty vague.